src/popover/popover.ts

```diff
@@ -204,10 +233,49 @@ export class NgbPopover implements OnInit, OnDestroy, OnChanges {
               this._elementRef.nativeElement, this._windowRef.location.nativeElement, this.placement,
               this.container === 'body'));
 
+      if (this.autoClose) {
+        this._ngZone.runOutsideAngular(() => {
+          let justOpened = true;
+          requestAnimationFrame(() => justOpened = false);
```

Comment:
I would add a 1-line comment for this, I don't think it will be obvious why we do it in a couple of months :)

Reply:
Agreed